tmt/cli.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Once more, IDE reformatting strikes again :)

Reply:
Oh, I've found the reason why the formatting is so different. `tmt` doesn't actually use `ruff-format` and `pep8` has no opinion on the formatting that either `ruff-format` or `pycharm` does